### PR TITLE
SC: Clean Log

### PR DESCRIPTION
**Pull request by KimKyungup**
## Proposed changes

This PR cleans logs like below. :)

INFO
- the things user should check.

DEBUG
- non regular things
- helpful things for debugging
- ignorable error

TRACE
- regular routine

WARN 
- the things cannot make system wrong.
- the things user can reference. 

ERROR
- the things can make system wrong.

## Types of changes

Please put an x in the boxes related to your change.

- [ ] Bugfix
- [ ] New feature or enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [x] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn/blob/master/CONTRIBUTING.md) doc
- [ ] I have signed the [CLA](https://cla-assistant.io/klaytn/klaytn)
- [ ] Lint and unit tests pass locally with my changes (`$ make test`)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Related issues

- Please leave the issue numbers or links related to this PR here.

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: node/sc/bridge_manager.go [hunk 1]
```diff
@@ -313,7 +313,7 @@ func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEv
 		uri, err = erc721.TokenURI(nil, ev.ValueOrTokenId)
 		if err != nil {
 			if err == vm.ErrExecutionReverted {
-				logger.Warn("The ERC721 token does not support URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
+				logger.Debug("Failed to get an ERC721 URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
```
Comment (aidan-kwon) [inline node/sc/bridge_manager.go]: "Failed" seems too strong for `Debug` level log. 
how about changing the message to "Unable to ~" or "Cannot get "?

Comment (KimKyungup) [reply node/sc/bridge_manager.go]: Good. I will reflect. :)

**Comment by KimKyungup** [comment]
@jeonghonah PTAL. :)

File: node/sc/bridge_manager.go [hunk 2]
```diff
@@ -313,7 +313,7 @@ func (bi *BridgeInfo) handleRequestValueTransferEvent(ev *RequestValueTransferEv
 		uri, err = erc721.TokenURI(nil, ev.ValueOrTokenId)
 		if err != nil {
 			if err == vm.ErrExecutionReverted {
-				logger.Warn("The ERC721 token does not support URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
+				logger.Debug("Unable to get an ERC721 URI", "erc721", ev.TokenAddress.String(), "onParent", bi.onChildChain, "tokenId", ev.ValueOrTokenId.String())
```
Comment (jeonghonah) [inline node/sc/bridge_manager.go]: How about `Trace()`?

Comment (KimKyungup) [reply node/sc/bridge_manager.go]: Good. I will modify this to `trace` if bridge contract is not mint/burn mode. :)

**Pull request merged**